src/website/src/pages/ec/components/navigation/inpage-navigation/docs/code.mdx

```diff
@@ -6,13 +6,4 @@ order: 2
 import { Playground, Html } from '@ecl/website-components';
 import inpageNavigationExample from '../demo';
 
-<Playground
-  system="ec"
-  selectedKind="components-navigation-inpage-navigation"
-  selectedStory="default"
-  frameHeight="350"
-  disableAutoResize={true}
-  showFrame
->
-  <Html markup={inpageNavigationExample} />
-</Playground>
+<Html markup={inpageNavigationExample} />
```

Comment:
Why this change? I don't mind having the html directly on the page if it works showing the spy behavior.

Reply:
Ouups !! Thks @kalinchernev this is a complete mistake, thank you for noticing it !!